src/modules/assignment/dto/update-assignment.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { IsNotEmpty } from 'class-validator';
+
+export class UpdateAssignmentDTO {
+  @IsNotEmpty()
+  readonly name: string;
```

Comment:
Also add links in dto

According to schema defined: https://github.com/codeforcauseorg/edu-server/blob/HEAD/src/schemas/assignment.schema.ts

Reply:
@Abhishek-kumar09 
1). removed @IsNotEmpty() decorator
2). added link object in updatedto (data transfer object)